scripts/installation/installROOT.sh

```diff
@@ -34,12 +34,11 @@ tar -xvzf $f
 rm -rf source
 mv root-$ROOT_VERSION source
 
-mkdir -p ${ROOT_DIR}/build
-cd ${ROOT_DIR}/build
+mkdir -p ${ROOT_DIR}/root_build
+cd ${ROOT_DIR}/root_build
 
-cmake -Wno-dev -Dbuiltin_glew=ON -DCMAKE_CXX_STANDARD=17 -Dgdml=ON -DCMAKE_INSTALL_PREFIX=${ROOT_DIR}/install  ../source
-make -j30
-make install
+cmake -DCMAKE_CXX_STANDARD=17 -DCMAKE_INSTALL_PREFIX=${ROOT_DIR}/install  ../source
+make -j8 install
```

Comment:
```suggestion
make -j install
```

Perhaps we can just us the `-j` option that uses the max. number of available threads, this way it can be independant of the system.

Reply:
While doing that my computer got stuck since it is going out of resources, perhaps we can check `nproc` and remove some cores?

Reply:
without a number it might take all the CPUs

Reply:
We might use: `cat /proc/cpuinfo | grep processor | wc -l`? And take half the number of CPUs?

Reply:
I updated my suggestion to use all minus 1 cores, I remember on some other project's documentation this is the recommended way, I have never seen half the number of cpu as recommended anywhere but we can also try it if this doesn't work.

Reply:
I have 12 cores in my WSL, but I already had issues using 10, so I think the safest is to go to the half of the cores.

Reply:
SO, cannot we do `($(nproc)/2)`

Reply:
So, `make -j8` is probably ok, perhaps even `make -j4`. Better to avoid problems.